csrc/cache_kernels.cu

```diff
@@ -435,6 +436,8 @@ void reshape_and_cache_flash(
   int key_stride = key.stride(0);
   int value_stride = value.stride(0);
   int block_stride = key_cache.stride(0);
+  int page_stride = key_cache.stride(1);
+  int head_stride = key_cache.stride(2);
```

Comment:
I think these should be int64. Shouldn't the `page_stride` be rolling over during testing?